content/common/cli-reference.md

```diff
@@ -685,13 +955,48 @@ SUBCOMMANDS:
 
 {{ blockEnd }}
 
-{{ blockEnd }}
+{{ startTab "v1.2.0"}}
 
-> Please note: Spin `help` is a convenient way to access help using a subcommand, instead of using the `--help` option. For example, `spin help cloud` will give you the same output as `spin cloud --help`. Similarly, `spin help build` will give you the same output as `spin build --help` and so forth.
+<!-- @selectiveCpy -->
 
-### Login
+```console
+$ spin help      
```

Comment:
This seems to entirely recapitulate `spin --help` from the top of the page.  Not sure why we would bother including it, especially when we haven't felt it was worth it in the past...?

Reply:
It is a fair point. The main reason `spin help` is in the CLI Reference is because it is listed as a subcommand (a matter of just referencing what exists in the terminal). These docs are generated by stepping through each of the commands and their subcommands and `spin help` happens in that process. As an aside, I kinda like the indented note that explains how the `subcommand` `help` can be used in place of an `option` and how that relationship exists with other commands i.e. `spin help build` (is the same as `spin build --help`).

Reply:
Oh yeah the note is grand it was just the re-printing of the top level help.  But I misread that this was a new thing - we did the same on 1.0 and 1.1 so carry on.

_slinks back into curmudgeon cave_